jobs/build-kola-containers.Jenkinsfile

```diff
@@ -7,6 +7,8 @@ node {
 
 properties([
     pipelineTriggers([
+        // run weekly to ensure regular updates
+        cron("H H */7 * *"),
```

Comment:
Hmm, shouldn't it be `H H * * H`?

But actually, we can use `@weekly` instead: https://www.jenkins.io/doc/book/pipeline/syntax/#cron-syntax

Reply:
updated!